packages/start/bin.cjs

```diff
@@ -519,7 +519,7 @@ async function resolveConfig({ configFile, root, mode, command }) {
     DEBUG('config file: "%s"', configFile);
   }
 
-  let config = await vite.resolveConfig({ mode, configFile, root }, command);
```

Comment:
This line, passing the mode to ViteJS, is what fixes the spa-test. It doesn't seem to need the removal of the adapter.